Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -355,6 +355,8 @@
     <string name="backend_multithreading_description">Enables graphics backend multithreading (Vulkan only). May affect performance. If unsure, leave this checked.</string>
     <string name="prefer_vs_for_point_line_expansion">Prefer VS for Point/Line Expansion</string>
     <string name="prefer_vs_for_point_line_expansion_description">On backends that support both using the geometry shader and the vertex shader for expanding points and lines, selects the vertex shader for the job. May affect performance.</string>
+    <string name="cpu_cull">Cull Vertices on the CPU</string>
+    <string name="cpu_cull_description">Cull vertices on the CPU to reduce the number of draw calls required. May affect performance. If unsure, leave this checked.</string>
```

Comment:
If the default is to disable it, the recommendation should be to `leave this unchecked`.